static/app/utils/url/validateLinkExists.ts

```diff
@@ -0,0 +1,20 @@
+import {t} from 'sentry/locale';
+import {handleXhrErrorResponse} from 'sentry/utils/handleXhrErrorResponse';
+
+/**
+ * Checks if a URL exists by sending a HEAD request.
+ * @param url - The URL to check.
+ * @returns A promise that resolves to `true` if the URL exists, or `false` otherwise.
+ */
+export async function validateLinkExists(url: string): Promise<boolean> {
+  try {
+    const response = await fetch(url, {method: 'HEAD'}); // Using HEAD method to just check if the resource exists
+    if (response.ok) {
+      return true;
+    }
+    return false;
+  } catch (error) {
+    handleXhrErrorResponse(t('Unable to validate if the link exists'), error);
```

Comment:
Do we really want to create an exception in Sentry in this case?
I would leave this to the calling code.